backend/assets/misc/device_names.txt

```diff
@@ -20,7 +20,7 @@ Kylo Renn
 Grievous
 Qui-Gon Jinn
 Dark Maul
-Rey
+Rey Palpatine
```

Comment:
There is a missing newline here isn't it ?

Reply:
Star Wars 9 reference 💀

Reply:
Oh right, forgot about that :sweat_smile: